docs/_deployments/aws-lambda.md

```diff
@@ -121,7 +121,7 @@ Update the [source code that imports your modules](https://github.com/slackapi/b
 const { App, AwsLambdaReceiver } = require('@slack/bolt');
 ```
 
-> 💡  If you are planning on implementing authentication with OAuth, as of today you need to use the [`ExpressReceiver`](https://github.com/slackapi/bolt-js/blob/main/src/receivers/ExpressReceiver.ts).
+> 💡  If you are planning on implementing authentication with OAuth, as of today you need to use the [`ExpressReceiver`](https://github.com/slackapi/bolt-js/blob/main/src/receivers/ExpressReceiver.ts). Additionally, when using other receivers with that are not the `AwsLambdaReceiver`, please note that the `processBeforeResponse: true` property is required to be passed in to your receiver initialization.
```

Comment:
```suggestion
> 💡  If implementing authentication with OAuth, you must use the [`ExpressReceiver`](https://github.com/slackapi/bolt-js/blob/main/src/receivers/ExpressReceiver.ts). Please note that when using `ExpressReceiver`, the `processBeforeResponse: true` property is required during initialization to avoid latency issues.
```

I had a look at the original conversation to guide the suggestion, but could have gotten it wrong. Feel free to ignore! As far as I could tell, the suggestion we make (and that the developer used) was `ExpressReceiver`, so I kept the language around that specific case.

Reply:
I think this makes it even more clear. Thank you @misscoded

Reply:
This is perfect! Thanks so much @misscoded 🙌